### DISCUSSION TITLE
Where is the "Archive" feature for completed tasks?

### DISCUSSION THREAD

**Discussion by TriggerDingus** [ransome1/sleek#713] [Q&A] [answered]
I only do archiving of completed tasks about every 6 months. Well, the time to do so has approached but I can't seem to find where the "Archive" feature is. I recall it was always easy to find but I've looked everywhere and can't find it anymore.  I pressed the "a" key since I saw on this forum that's supposed to trigger it, but it doesn't on mine. I pressed the Arrow/Paper icon 
![2024-06-01_14-48](https://github.com/ransome1/sleek/assets/75340371/fc1bbe1c-6b04-4562-bc6b-883de4de9155)
 on the menu (which I never use - probably because I open my ToDo file once and that's it) and it looks like it just opens a new file but doesn't archive to one.  Any guidance would be appreciated as to how to initiate the archive feature!

**Comment by TriggerDingus** [comment]
Anyone out there?

**Reply by ransome1** [reply]
![Screenshot 2024-06-04 at 6 23 43 PM](https://github.com/ransome1/sleek/assets/11188741/542dddd2-dcac-426b-8d6c-16b5f8568dd0)

To trigger archiving you can either press the archiving button or use the keyboard shortcut `Ctrl + Alt + A`.

**Reply by TriggerDingus** [reply]
My Archive icon is missing after all! Here is a screen capture:
![2024-06-04_12-53](https://github.com/ransome1/sleek/assets/75340371/0b9f6541-43ff-4470-9397-19d6ac0f2176)

I'm on Sleek 2.0.13 on Mint via Flatpak.

**Reply by ransome1** [reply]
Is it possible that you dont have any completed items? The archiving is only available, when there is at least one completed item.

**Reply by TriggerDingus** [reply]
> Is it possible that you dont have any completed items? The archiving is only available, when there is at least one completed item.

I found the issue based upon your comment. I have hundreds, perhaps a thousand or more completed items. The issue preventing the Archive icon from being displayed is that I keep them hidden using the filter since I don't want to have them cluttering up an already large ToDo list. Once I display them, the Archive icon is visible. I know it's a judgment call to have the icon displayed or not based upon this filter - my vote is to have the Archive icon always displayed so the user knows the feature is there and an info box could be displayed if they press it with nothing to archive.  Since I archive only every few months, I'll always forget how to get that box back when I need to archive again.

Here's the filter that enables/disables the Archive icon for me:
![2024-06-04_16-34](https://github.com/ransome1/sleek/assets/75340371/991dc534-e720-43b2-9720-ed3a55bba335)

**Reply by ransome1** [reply]
Could you check the latest pre-release please and tell me if this happens here as well? https://github.com/ransome1/sleek/releases/tag/v2.0.14-rc.1

I vaguely remember some minor bug, which could have lead to the behavior you are experiencing. And furthermore I think we solved this during recent refactoring work, which is part of the latest pre-release.

**Reply by TriggerDingus** [reply]
PS - My compliments on Sleek. I've been using it a few years now and it's a very smartly designed app that continues to evolve in thoughtful and effective ways. I've been using To Do systems for decades, including microsoft outlook, Lotus Notes, etc and their task functionality doesn't touch Sleek.  It's very well done.

**Reply by TriggerDingus** [reply, marked as answer]
> Could you check the latest pre-release please and tell me if this happens here as well? https://github.com/ransome1/sleek/releases/tag/v2.0.14-rc.1
> 
> I vaguely remember some minor bug, which could have lead to the behavior you are experiencing. And furthermore I think we solved this during recent refactoring work, which is part of the latest pre-release.

I'll test this today and let you know.

**Reply by TriggerDingus** [reply]
Fixed in pre-release.
